pytorch_toolkit/action_recognition/README.md

```diff
@@ -191,4 +191,4 @@ mo.py --input_model resnet34_vtn.onnx --input_shape '[1,16,3,224,224]'
 
 ## Demo
 
-You can try your models after converting them to the OpenVINO™ format or a [pretrained model from OpenVINO™](https://docs.openvinotoolkit.org/latest/usergroup10.html) using the [demo application from OpenVINO™ toolkit](https://docs.openvinotoolkit.org/latest/_demos_python_demos_action_recognition_README.html)
+You can try your models after converting them to the OpenVINO™ format or a [pretrained model from OpenVINO™](https://docs.openvinotoolkit.org/2020.4/usergroup10.html) using the [demo application from OpenVINO™ toolkit](https://docs.openvinotoolkit.org/2020.4/_demos_python_demos_action_recognition_README.html)
```

Comment:
Are the links supposed to point to 2020.4 version?
To refer to the latest version, please use the following links:
https://docs.openvinotoolkit.org/latest/text_detection.html
https://docs.openvinotoolkit.org/latest/omz_demos_python_demos_action_recognition_README.html

Reply:
I see. 

So the 
https://docs.openvinotoolkit.org/2020.4/usergroup13.html 
is actually 
https://docs.openvinotoolkit.org/latest/text_detection.html

and 

https://docs.openvinotoolkit.org/2020.4/_demos_python_demos_action_recognition_README.html
is 
https://docs.openvinotoolkit.org/latest/omz_demos_python_demos_action_recognition_README.html

Sorry for the confusion. @abhatikar